controllers/data.controller.js

```diff
@@ -11,12 +15,22 @@ exports.getTopics = (req, res, next) => {
 };
 
 exports.getArticlesById = (req, res, next) => {
-  const id = req.params.articles_id;
-  fetchArticles(id)
+  const id = req.params.article_id;
+  fetchArticlesById(id)
     .then((article) => {
       res.status(200).send(article);
```

Comment:
I know its not this ticket - but you should be sending them back in objects i.e. {article}